kafka.go

```diff
@@ -27,6 +30,38 @@ func NewKafkaProducer(logger *log.Logger, stats *Stats, config *Config) (NozzleP
 	// TODO (tcnksm): Enable to configure more properties.
 	producerConfig := sarama.NewConfig()
 
+	if !config.Kafka.InsecurePlainText {
+		if len(config.Kafka.CACerts) == 0 {
+			return nil, errors.New("please specify at least one ca_certificate, else set insecure_use_plaintext")
```

Comment:
is specifying additional CA certs really required? what if the kafka cert is already signed by a trusted CA?

Reply:
I agree it would be sensible to use the system trusted roots if no CA certs are specified - I'll update the PR to do so (am on holiday break now, so will be likely be in a few weeks).

Reply:
Changed to default to system CA pool if no CAs are specified.